msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,306 @@
+### YamlMime:Tutorial
+title: Teams File Upload Bot test demo
+metadata:
+  title: Teams File Upload Bot test demo
+  description: In this tutorial, you'll learn how to upload files to Teams from a bot and how to receive a file sent to a bot as an attachment.
+  audience: Developer
+  level: Beginner
+  ms.date: 24/12/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: bots/how-to/authentication/auth-aad-sso-bots.md
+  nextTutorialTitle: Read more on How Microsoft Teams bots work
+  ms.custom: mvc
+  ms.localizationpriority: none
+items:
+- durationInMinutes: 1
+  content: |
+   This bot has been created using Bot Framework, it shows how to upload files to Teams from a bot and how to receive a file sent to a bot as an attachment. 
```

Comment:
The bot is created using Bot Framework. It shows how

Reply:
Changes are made.